### PR TITLE
Add abstract

### PR DESCRIPTION
**Pull request by SarahMalykke**
Hey @jessb0t 🙂 I added my abstract for the NCUR poster in the materials folder within putt-putt-dataset, please let me know if you want it somewhere else

### CODE REVIEW

**Comment by jessb0t** [comment]
@SarahMalykke Should this PR be into `dev` instead of `main`?

**Comment by SarahMalykke** [comment]
Ah yes! Should be changed now 🙂 @jessb0t

File: materials/abstract_ncur.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
... observation. Yet, these studies are limited in showing the effects of social observation within more ecologically valid, “real world” settings. Our current study seeks to test whether error-related brain activity, while under social observation, differs between socially anxious and non-anxious participants within an ecologically valid task context. Towards this end, we will assess error-related brain activity via Electroencephalography (EEG) during a real-world sports task (golfing), as well as via a more traditional, computer-based task. By employing an ecologically valid sports task, this study not only has the potential to translate prior laboratory-based research into the applied, real-world domain, but also has potential implications for understanding “choking under pressure” in the context of sports. Data will be analyzed using MATLAB/ R.. We hypothesize that individuals with social anxiety will exhibit elevated error-related brain activity in the presence of others, whether on a traditional lab-based computer task or within an ecologically-valid sports task. Moreover, we hypothesize that the ecological valid task will correlate more strongly with measures of social anxiety.
```
Comment (jessb0t) [inline materials/abstract_ncur.md:10]: A few minor corrections:
- "electroencephalography" should not be capitalized
- "ecological valid" in the last sentence should be corrected to "ecologically-valid"
- inconsistency in whether "ecologically-valid" has a hyphen; either is fine, as long as we are consistent

**Comment by jessb0t** [comment]
@SarahMalykke A few minor edits mentioned above.  I'm approving now so you can merge to `dev` once you have made those changes and then PR to `main` (tagging George on that one). :)

**Review by jessb0t — APPROVED** [review]

**Pull request merged**